source/uwp/Renderer/lib/AdaptiveChoiceSetInput.h

```diff
@@ -26,14 +26,14 @@ namespace AdaptiveNamespace
 
```

Comment:
feel free to punt to a new issue if you like. because this is passed by reference, `_In_` is redundant (see `SAL.2` in internal coding guidelines).

you can find other instances with this handy command line: `findstr /spin "runtimeclassinitialize" * | findstr /c:"_In_" | findstr /c:"&"`

Reply:
Thanks for the query!

---
In reply to: [244876216](https://github.com/Microsoft/AdaptiveCards/pull/2267#discussion_r244876216) [](ancestors = 244876216)